### PR TITLE
Update to 3.11.10 and fix the build

### PR DESCRIPTION
**Pull request by Anthchirp**
Currently the build is broken. Have a go at fixing it.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Anthchirp** [comment]
@conda-forge-admin please rerender

**Comment by Anthchirp** [comment]
@conda-forge-admin please rerender

**Comment by Anthchirp** [comment]
This is ready to be merged. I've added myself as maintainer. Feel free to take me off if not desired.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
+    mechanisms for domain-decomposition of the atoms.
 
-  doc_url: https://wiki.fysik.dtu.dk/asap/asap
+  doc_url: https://wiki.fysik.dtu.dk/asap
   dev_url: https://gitlab.com/asap/asap
 
 extra:
   recipe-maintainers:
+    - Anthchirp
     - CJ-Wright
```
Comment (CJ-Wright) [inline recipe/meta.yaml:56]:
```suggestion
```

**Comment by CJ-Wright** [comment]
I took the liberty of removing myself. Thank you for taking over!

**Pull request merged**